receiver/filelogreceiver/README.md

```diff
@@ -111,4 +111,66 @@ receivers:
           parse_from: attributes.time
           layout: '%Y-%m-%d %H:%M:%S'
 ```
+## Example - Using filelog receiver with [ADOT](https://github.com/aws-observability/aws-otel-collector) (AWS Distro for OpenTelemetry)
+DISCLAIMER: ADOT doesn't officially support the filelog receiver yet, so the below example is a temporary workaround to get it working
```

Comment:
This is the contrib repository, so no mentions to ADOT are necessary.

Reply:
Please remove all mentions to ADOT. All the configurations refer to this component, therefore all references should be towards this component alone. 

E.g.:  This example will show how to get the filelog receiver working.

Reply:
Seconded. This documentation concerns this receiver, not ADOT